src/openassetio-core-c/include/openassetio/c/InfoDictionary.h

```diff
@@ -49,6 +53,9 @@ extern "C" {
 #define oa_InfoDictionary_setFloat OPENASSETIO_NS(InfoDictionary_setFloat)
 #define oa_InfoDictionary_setStr OPENASSETIO_NS(InfoDictionary_setStr)
 
+/// @}
+// oa_InfoDictionary_alias
```

Comment:
[fix] `oa_InfoDictionary_aliases` elsewhere

Reply:
Fixed in 2b654545